CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 ## UNRELEASED
 
+## v0.103.1
+
+:new: What's new:
+- Add relative ./ path support for MD images (#6057)
+- UI: Support uploading multiple files and directories (#6074)
+- UI: show summary of uploaded files (#6079)
+
+:bug: Bugs fixed:
+- Docs: fix lakefs_object and invalid sql statements (#6072)
+- Delta Diff fails on certain operations (#6065)
+- Fix: URI parse with whitespace (#6080)
```

Comment:
Might want to preface with the affected component.

```suggestion
- Fix (HadoopFS): URI parse with whitespace (#6080)
```

Reply:
Thanks, another catch: HadoopFS is also detached from version releases...